### PR TITLE
More extpos

### PR DESCRIPTION
**Pull request by AngledLuffa**
Several improvements to the ExtPos labeling or otherwise the annotations of fixed expressions in PUD

### CODE REVIEW

**Comment by AngledLuffa** [comment]
@nschneid 

addresses more of https://github.com/UniversalDependencies/UD_English-PUD/issues/64

File: en_pud-ud-test.conllu [hunk 1]
```diff
@@ -5404,8 +5404,8 @@
 21	photographs	photograph	NOUN	NNS	Number=Plur	17	obl	17:obl:in|23:nsubj	_
 22	that	that	PRON	WDT	PronType=Rel	23	nsubj	21:ref	_
 23	looked	look	VERB	VBD	Mood=Ind|Tense=Past|VerbForm=Fin	21	acl:relcl	21:acl:relcl	_
-24	as	as	SCONJ	IN	_	30	case	30:case	_
-25	if	if	SCONJ	IN	_	24	fixed	24:fixed	_
+24	as	as	SCONJ	IN	_	30	mark	30:mark	_
+25	if	if	SCONJ	IN	_	30	mark	30:mark	_
```
Comment (nschneid) [inline en_pud-ud-test.conllu:5408]: This is documented as `fixed`: https://universaldependencies.org/en/dep/fixed.html#pragmaticmiscellaneous-markers

File: en_pud-ud-test.conllu [hunk 2]
```diff
@@ -7210,8 +7210,8 @@
 11	that	that	PRON	WDT	PronType=Rel	12	nsubj	8:ref	_
 12	makes	make	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	acl:relcl	8:acl:relcl	_
 13	it	it	PRON	PRP	Case=Acc|Gender=Neut|Number=Sing|Person=3|PronType=Prs	12	obj	12:obj|17:nsubj:xsubj	_
-14	all	all	DET	PDT	PronType=Tot	16	advmod	16:advmod	_
-15	the	the	DET	DT	Definite=Def|PronType=Art	14	fixed	14:fixed	_
+14	all	all	ADV	RB	_	16	advmod	16:advmod	_
+15	the	the	DET	DT	Definite=Def|PronType=Art	16	det	16:det	_
```
Comment (nschneid) [inline en_pud-ud-test.conllu:7214]: Agree it's not `fixed` but I don't know the right analysis. GUM and EWT appear inconsistent on whether "the" attaches to "more" or the adjective: https://universal.grew.fr/?custom=677004c24dba9. This is related to the comparative correlative, which we have not developed a standard for.

File: en_pud-ud-test.conllu [hunk 3]
```diff
@@ -20181,9 +20181,9 @@
 7	first	first	ADJ	JJ	Degree=Pos|NumForm=Word|NumType=Ord	8	amod	8:amod	_
 8	hour	hour	NOUN	NN	Number=Sing	4	obl	4:obl:in	_
 9	is	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	cop	13:cop	_
-10	more	more	ADV	RBR	_	13	advmod	13:advmod	_
-11	or	or	CCONJ	CC	_	10	fixed	10:fixed	_
-12	less	less	ADV	RBR	Degree=Cmp	10	fixed	10:fixed	_
+10	more	more	ADV	RBR	Degree=Cmp	13	advmod	13:advmod	_
+11	or	or	CCONJ	CC	_	12	cc	12:cc	_
+12	less	less	ADV	RBR	Degree=Cmp	10	conj	10:conj:or|13:advmod	_
```
Comment (nschneid) [inline en_pud-ud-test.conllu:20186]: "more or less equal" - this has the idiomatic meaning of "approximately equal". But "more or less" is not documented in the fixed list so I agree with this change.

**Review by nschneid — CHANGES_REQUESTED** [review]

File: en_pud-ud-test.conllu (hunk 1 above)
Comment (AngledLuffa) [reply en_pud-ud-test.conllu:5408]: That's fair, but in EWT it's treated as `mark mark` as I did here.  In GUM it's labeled `mark fixed` but with no `ExtPos`.  What should be the appropriate `ExtPos` - `SCONJ`?

File: en_pud-ud-test.conllu (hunk 2 above)
Comment (AngledLuffa) [reply en_pud-ud-test.conllu:7214]: Consider it an incremental improvement for now, and another "if we remember it" for if that standard happens, kind of like `nmod:desc` was?

File: en_pud-ud-test.conllu (hunk 1 above)
Comment (nschneid) [reply en_pud-ud-test.conllu:5408]: Oh indeed, EWT does not follow that guideline. Interesting. Yes, ExtPos=SCONJ if it's fixed.

Comment (AngledLuffa) [reply en_pud-ud-test.conllu:5408]: got it, so, SCONJ / ADP with ExtPos=SCONJ?  That will be more like GUM, assuming they add the ExtPos

Comment (amir-zeldes) [reply en_pud-ud-test.conllu:5408]: Yes, I can do that, np.

**Pull request merged**